services/user/Tokens/ui/src/hooks/network/useLoggedInUser.ts

```diff
@@ -0,0 +1,27 @@
+import { supervisor } from "@/main";
+import { useQuery } from "@tanstack/react-query";
+import { toast } from "sonner";
+import { z } from "zod";
+
+let toasted = false;
+
+export const useLoggedInUser = () =>
+  useQuery({
+    queryKey: ["loggedInUser"],
+    queryFn: async () => {
+      if (!toasted) {
+        toast("Fetching account...");
+      }
+      const res = await supervisor.functionCall({
+        method: "getLoggedInUser",
```

Comment:
This should now be `getCurrentUser` (in the latest main).
It's also no longer restricted only to the active app, so the interface changed from `activeApp` to just "api"